sources/Inputs/src/NetworkManager.cpp

```diff
@@ -100,5 +100,11 @@ NetworkManager::walkNodes() {
   }
 }
 
+boost::optional<boost::shared_ptr<Node>>
+NetworkManager::getSlackNode() const {
+  // for now, we cannot retrieve from dynawo data interface this information: we will in a future version
```

Comment:
Please add a todo so that we can retrieve easily open subjects

Reply:
OK